src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const populationElements = document.querySelectorAll('span.population');
+  const populations = Array.from(populationElements).map((el) => {
+    const num = parseFloat(el.textContent.replace(/,/g, ''));
+
+    return isNaN(num) ? 0 : num;
+  });
+
+  const totalPopulation = populations.reduce((acc, num) => acc + num, 0);
+  const averagePopulation = totalPopulation / populations.length;
+
+  const formatNumber = (num) => num.toLocaleString();
+
+  document.querySelector('span.average-population').textContent =
```

Comment:
Ensure that the HTML element with the class 'average-population' exists in the DOM. If it doesn't, this line will cause a runtime error.